src/applications/disability-benefits/2346/containers/OrderHistory.jsx

```diff
@@ -0,0 +1,228 @@
+import ErrorableCheckbox from '@department-of-veterans-affairs/formation-react/ErrorableCheckbox';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { focusElement } from 'platform/utilities/ui';
+import {
+  getDLCAccessoriesData,
+  getDLCBatteryData,
+  // NOTE: Decision was to take socks out of MVP -@maharielrosario at 1/28/2020, 9:49:20 AM
+  // getDLCSocksData,
+  updateDLCData,
+} from '../actions';
+
+class OrderHistory extends Component {
```

Comment:
It looks like we're using different syntax for components.. probably should pick either ```extends Component``` or ```extends React.Component```

Reply:
Yeah we will stick with `React.Component` for now. And this page will be totally redone. And is not expected to be delivered into 3 sprints from now.